docs/GettingStarted/configuration/PelorusExporters.md

```diff
@@ -1,175 +1,326 @@
+    - [Commit time](ExporterCommittime.md#commit-time-exporter-configuration-options)
+    - [Failure](ExporterFailure.md#failure-time-exporter-configuration-options)
 
-To store a personal access token securely and to make the token available to Pelorus use
-Openshift secrets. Pelorus can utilize secrets for all of the exporters. The following
-is an example for the committime exporter.
+### enabled
 
-A simple Github example:
-```shell
-oc create secret generic github-secret --from-literal=TOKEN=<personal access token> -n pelorus
-```
+- **Required:** no
+    - **Default Value:** true
+- **Type:** boolean
 
-A Pelorus exporter can require additional information to collect data such as the 
-remote `GIT_API` or `API_USER` information.  It is recommended to consult the requirements
-for each Pelorus exporter in this guide and include the additional key / value information in the Openshift secret. An API example is `github.mycompany.com/api/v3`
-or `https://gitea.mycompany.com`.
+    If set to `false`, the exporter is not deployed.
 
-The cli commands can also be substituted with Secret templates. Example files can be found [here](https://github.com/konveyor/pelorus/tree/master/charts/pelorus/secrets)
+### custom_certs
 
-Create a secret containing your Git username, token, and API example:
+- **Required:** no
+- **Type:** list
 
-```shell
-oc create secret generic github-secret --from-literal=API_USER=<username> --from-literal=TOKEN=<personal access token> --from-literal=GIT_API=<api> -n pelorus
-```
+    List of `map_name`s, like in the following example.
 
-A Jira example:
-```shell
-oc create secret generic jira-secret \
---from-literal=SERVER=<Jira Server> \
---from-literal=API_USER=<username/e-mail> \
---from-literal=TOKEN=<personal access token> \
--n pelorus
-```
+    ```yaml
+    custom_certs:
+      - map_name: name
+    ```
 
-A ServiceNow example:
-```shell
-oc create secret generic snow-secret \
---from-literal=SERVER=<ServiceNow Server> \
---from-literal=API_USER=<username> \
---from-literal=TOKEN=<personal access token> \
---from-literal=TRACKER_PROVICER=servicenow \
---from-literal=APP_FIELD=<Custom app label field> \
--n pelorus
-```
+    Check [Custom Certificates](#custom-certificates) for more information.
+
+### image_tag
+
+- **Required:** no
+    - Only applicable for development configuration, **do not use in production**
+    - **Default Value:** stable
+- **Type:** string
+
+    Used to set exporter image tag (or custom image, if [image_name](#image_name) is set).
+
+    Check [Development guide](../../Development.md) for more information.
+
+### image_name
+
+- **Required:** no
+    - Only applicable for development configuration, **do not use in production**
+- **Type:** string
+
+    Used to deploy exporter with the user built images or pre-built images mirrored in other than [quay.io](https://quay.io/) registry. If no tag is specified in the image name, [image_tag](#image_tag) is used.
```

Comment:
That is not true, it *can* be also quay.io.

Used to deploy exporter with the user built images or pre-built images hosted in non default container image registry. The container image URI may be with or without `:tag` suffix. If no tag suffix is specified in the URI, [image_tag](#image_tag) is used.

Reply:
changed